### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by callunity**
* removes link to the outdated admin manual (deploys from master)
* updates the release/upgrade procedure to specify where to find the latest release
* highlights the user tutorial

### CODE REVIEW

**Comment by callunity** [comment]
Think it's worth updating the date of the admin manual to today's date?

File: docs/_posts/2016-01-11-admin-manual.md [hunk 1]
```diff
@@ -115,14 +115,18 @@ and populate it with the following information:
       username="tgaclims"
       password="tgaclims"/>
       <Parameter name="miso.propertiesFile" value="file:${catalina.home}/conf/Catalina/localhost/miso.properties" override="false"/>
-      <Parameter name="miso.name" value="Test"/>
+      <Parameter name="miso.instanceName" value="Your MISO instance name"/>
+      <!-- uncomment if using Runscanner -->
```
Comment (apmasell) [inline docs/_posts/2016-01-11-admin-manual.md]: I don't think this belong in this file.

File: docs/_posts/2016-01-11-admin-manual.md [hunk 2]
```diff
@@ -115,14 +115,18 @@ and populate it with the following information:
...
+      <!--
+      <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
```
Comment (djcooke) [inline docs/_posts/2016-01-11-admin-manual.md]: It looks like this is only used by RunScanner itself. Is there a reason MISO needs to know about it?

If so, it's not in our stage or production config...

File: docs/_posts/2016-01-11-admin-manual.md [hunk 3]
```diff
@@ -115,14 +115,18 @@ and populate it with the following information:
...
+      -->
     </Context>
 
 Make sure the database path in `ROOT.xml` is correct for your install:
 
     url="jdbc:mysql://your.database.server:3306/lims"
 
-Also, set the `miso.name` parameter to something to help distinguish testing
+Also, set the `miso.instanceName` parameter to something to help distinguish testing
```
Comment (djcooke) [inline docs/_posts/2016-01-11-admin-manual.md]: `miso.instanceName` comes from `miso.properties` now. Should be removed from `ROOT.xml`

**Comment by callunity** [comment]
`runscanner.configFile` does belong in the MISO `ROOT.xml` if runscanner is being hosted on the same Tomcat as MISO. I've updated the file to clarify this.

**Pull request merged**